package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 0",
+    "test": "concurrently -k -p \"[{name}]\" -n \"Backend,Frontend\" -c \"green.bold,yellow.bold\" \"npm run test:backend\" \"npm run test:frontend\"",
+    "test:frontend": "cd frontend && npm run test",
+    "test:backend": "cd backend && npm run test",
```

Comment:
Can we also have `test:shared`?

Reply:
Resolved in [647da5c](https://github.com/opengovsg/postmangovsg/pull/1367/commits/647da5c9157bb861a5c7310a2849f96d2602009e)